lms/djangoapps/experiments/views.py

```diff
@@ -82,3 +81,25 @@ def bulk_upsert(self, request):
 
             serializer = self.get_serializer(upserted, many=True)
             return Response(serializer.data)
+
+
+class ExperimentKeyValueViewSet(viewsets.ModelViewSet):
+    authentication_classes = (JwtAuthentication, SessionAuthenticationAllowInactiveUser,)
```

Comment:
`JwtAuthentication` is OAuth 2.0. We will retrieve the JWT from the access token endpoint, and pass it in the Authentication header for calls to the API.

See https://openedx.atlassian.net/wiki/display/OpenDev/OAuth+2.0.